src/Blocks/BlockFarmland.h

```diff
@@ -76,7 +101,9 @@ class cBlockFarmlandHandler final :
 			}
 			default:
 			{
-				a_Chunk.SetBlock(a_RelPos, E_BLOCK_DIRT, 0);
+				auto World = a_Chunk.GetWorld();
+				auto AbsPos = a_Chunk.RelativeToAbsolute(a_RelPos);
```

Comment:
Since our bounding box is a single block large, there's no need to get the world, just pass in the chunk here and call ForEachEntityInBox on the chunk instead.

Reply:
Done